### ISSUE TITLE
v2.0.0 release: translations!

### ISSUE THREAD

**Issue by sethvincent** [sethvincent/javascripting#85]
I'd like to get the translations released to npm, but it looks like there are a couple things we should settle first:

- #84 translate troubleshooting.md to `ko` & `es`
- #83 have an approach for lessons that are not yet translated

Is there anything else that should be included?

**Comment by marocchino** [comment]
I made a miss file list with file diffing.

  - problems/function-return-values/problem_es.md
  - problems/function-return-values/solution_es.md
  - problems/object-keys/problem_es.md
  - problems/scope/problem_es.md
  - problems/scope/solution_es.md
  - problems/this/problem_es.md
  - problems/this/solution_es.md
  - problems/function-return-values/problem_zh-cn.md
  - problems/function-return-values/solution_zh-cn.md
  - problems/object-keys/problem_zh-cn.md
  - problems/object-keys/solution_zh-cn.md
  - problems/scope/problem_zh-cn.md
  - problems/scope/solution_zh-cn.md
  - problems/this/problem_zh-cn.md
  - problems/this/solution_zh-cn.md
  - troubleshooting_es.md
  - troubleshooting_ko.md

**Comment by sethvincent** [comment]
nice! good idea @marocchino

**Comment by SomeoneWeird** [comment]
We should at least handle not having these files available gracefully, once we get more languages translated it's going to be near impossible to keep up and have everything translated before we release.

**Comment by hquan** [comment]
@marocchino I will add Chinese version of the missing files later. Thank you!

**Comment by martinheidegger** [comment]
Usually we didn't merge translations of the workshoppers until the respective persons thought of them as properly translated, that would mean dropping the unfinished translations and add a pull request for the chapters.

**Comment by a0viedo** [comment]
@thehazzard and @pepoviola are working on:
>  problems/scope/problem_es.md
 problems/scope/solution_es.md

the other missing spanish files are covered in https://github.com/sethvincent/javascripting/pull/94

**Comment by a0viedo** [comment]
Now that https://github.com/sethvincent/javascripting/pull/99 has landed, we're all set for this release.

@sethvincent @marocchino let me know if can help with anything else. Also, it would be great to have this on npm before the NodeSchool International Day :smiley:

**Comment by sethvincent** [comment]
Nice! I'll release a version now so we can test it and make fixes if needed before international day.

**Comment by theHazzard** [comment]
great! I will help with testing as soon it hits npm!

**Comment by sethvincent** [comment]
Published as v2.0.0.

@martinheidegger & @marocchino it looks like workshopper-adventure needs a korean translation.

**Comment by marocchino** [comment]
I [PRed](https://github.com/martinheidegger/workshopper/commit/281923360ef0389218f9f47fbc24159930b62605) it already. 

For now, You can use that like this.
```
"workshopper-adventure": "git://github.com/martinheidegger/workshopper.git#adventure",
```
:3 :3

**Comment by sethvincent** [comment]
Oh! Cool.

**Comment by sethvincent** [comment]
ok. released another version!

**Comment by marocchino** [comment]
:tada:

**Comment by martinheidegger** [comment]
Sorry for the delay. Since there was no response in https://github.com/workshopper/organizers/issues/3 I created and published https://github.com/workshopper/workshopper-adventure also i invited several of the members here to be contributes.

**Comment by SomeoneWeird** [comment]
Awesome :)

**Comment by marocchino** [comment]
I close this because we release 2.0 already. :dancers:

**Comment by a0viedo** [comment]
@marocchino
> "workshopper-adventure": "git://github.com/martinheidegger/workshopper.git#adventure",

I know we are still celebrating International Day, but just wanted to point out that this requires Git to be installed in Windows for the installation to work. I think we should avoid that as much as we can.

**Comment by marocchino** [comment]
@a0viedo Thanks. I did not know that. Hopefully, We can remove that when newer workshopper-adventure come.
/cc @martinheidegger can you release it, right?

**Comment by sethvincent** [comment]
i agree, that was an annoying thing for a few folks in seattle today.

it looks like we can install the current version of it from here now: https://github.com/workshopper/workshopper-adventure

**Comment by marocchino** [comment]
Umm.. but, that repo did not contain Korean translation. -3-

**Comment by sethvincent** [comment]
just released javascripting@2.0.3 to include the updated workshopper-adventure@3.4.5
